examples/seq2seq/seq2seq.py

```diff
@@ -161,8 +161,12 @@ def main():
                         help='GPU ID (negative value indicates CPU)')
     parser.add_argument('--resume', '-r', default='',
                         help='Resume the training from snapshot')
-    parser.add_argument('--unit', '-u', type=int, default=1024,
+    parser.add_argument('--unit', '-u', type=int, default=512,
```

Comment:
I copied from translate.py example in TensorFlow for comparison. https://github.com/tensorflow/models/blob/master/tutorials/rnn/translate/translate.py#L56